### ISSUE TITLE
Status check panics with traceback when container is in pending state

### ISSUE THREAD

**Issue by mukundansundar** [dapr/cli#609]
## Expected Behavior

Status check gracefully fails on pending errors. Upgrade stops when containers are in pending state.

## Actual Behavior

```
  NAME                   NAMESPACE    HEALTHY  STATUS                       REPLICAS  VERSION  AGE  CREATED
panic: runtime error: index out of range [0] with length 0

goroutine 65 [running]:
github.com/dapr/cli/pkg/kubernetes.(*StatusClient).Status.func1(0xc000465e90, 0x2c455c0, 0xc000878f20, 0xc000465ea0, 0xc0007a0e60, 0x298aef5, 0xe)
	/Users/runner/work/cli/cli/pkg/kubernetes/status.go:98 +0x9cf
created by github.com/dapr/cli/pkg/kubernetes.(*StatusClient).Status
	/Users/runner/work/cli/cli/pkg/kubernetes/status.go:70 +0x15a
```

## Steps to Reproduce the Problem

Have some dapr containers in pending state, ie not enough resource in k8s cluster. run `dapr status -k`

## Release Note

RELEASE NOTE: Fix status check panic on pending containers

**Comment by 1046102779** [comment]
solved. [dapr status -k](https://github.com/dapr/cli/pull/617)

**Comment by mukundansundar** [comment]
closed by #617
